README.md

```diff
@@ -90,196 +67,133 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+<br>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Video Presentation 
+[Video Demo Link](https://drive.google.com/drive/u/0/folders/1BUFwAkKcI3iUr-kRtgKTpWBJNlCVFNui)
```

Comment:
- Please provide an easier way to access the video. Kindly make the link available and easy to access to video

Reply:
@OLIPLICHE Thanks for your review and comments.  

I have uploaded the video to youtube, attach the link in the readme file.

Reply:
This change has been done well